apps/gauzy/src/app/@theme/components/theme-sidebar/theme-settings/theme-settings.component.ts

```diff
@@ -1,22 +1,40 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, AfterViewChecked } from '@angular/core';
 import { NbSidebarService } from '@nebular/theme';
+import { tap } from 'rxjs/operators';
 
 @Component({
 	selector: 'ngx-theme-settings',
 	styleUrls: ['./theme-settings.component.scss'],
 	templateUrl: './theme-settings.component.html'
 })
-export class ThemeSettingsComponent implements OnInit, OnDestroy {
+export class ThemeSettingsComponent implements OnInit, AfterViewChecked, OnDestroy {
+
+	private state: boolean;
 
 	constructor(
 		private readonly sidebarService: NbSidebarService
-	) {}
+	) { }
+
+	ngOnInit(): void { }
 
-	ngOnInit(): void {}
+	ngAfterViewChecked(): void {
+		this.sidebarService
+			.getSidebarState('settings_sidebar')
+			.pipe(
+				tap(
+					(state) =>
+						(this.state = state === 'expanded' ? true : false)
+				),
```

Comment:
@Gramosx  You must unsubscribe the subscription to prevent memory leaks.
```suggestion
				),
				 untilDestroy(this)
```